.github/workflows/package.yml

```diff
@@ -200,12 +194,28 @@ jobs:
           ls -Rl
         working-directory: output_tgz
 
+      - name: compute SDK hash for tgz files
+        shell: bash
+        run: |
+          tgz_files_list=$(find -type f -name '*.tgz')
+          for tgz_file in "${tgz_files_list[@]}"; do
+            ${{ env.hashCommand }} --tag ${tgz_file} >> ../firebase_unity_sdk_hash.txt
+            echo "::warning ::$(cat firebase_unity_sdk_hash.txt)"
```

Comment:
Should the message be clearer on what it is doing?

Reply:
updated